app/models/conversion_host.rb

```diff
@@ -36,9 +48,9 @@ def source_transport_method
   end
 
   def ipaddress(family = 'ipv4')
-    return address if address && IPAddr.new(address).send("#{family}?")
+    return address if address.present? && IPAddr.new(address).send("#{family}?")
```

Comment:
darn - doesn't seem like it should be necessary. (but I believe you that it is)

Reply:
May not be, I was fighting with some other things, and thought it would help. Doesn't hurt, so I'll leave it.